openproblems/tasks/dimensionality_reduction/methods/tsne.py

```diff
@@ -15,8 +15,9 @@
     code_version=check_version("MulticoreTSNE"),
     image="openproblems-python-extras",
 )
-def tsne(adata, test=False, n_pca=50):
```

Comment:
Question: why did we remove these methods rather than adding HVG as an additional method?

Reply:
I just renamed them when we changed the preprocessing function. Happy to have other variants, just need to decide what preprocessing they should have.

Reply:
I think it's fine as is.